.circleci/config.yml

```diff
@@ -45,14 +45,34 @@ jobs:
       - install-bazel-linux-rbe
       - checkout
       - run: bazel run @graknlabs_build_tools//checkstyle:test-coverage
-      - run: |
-          SONARCLOUD_CODE_ANALYSIS_CREDENTIAL=$SONARCLOUD_CREDENTIAL \
-            bazel run @graknlabs_build_tools//sonarcloud:code-analysis -- \
-            --organisation $SONARCLOUD_ORGANISATION --project-key $SONARCLOUD_PROJECT \
-            --commit-id=$CIRCLE_SHA1 --branch=$CIRCLE_BRANCH
       - run-bazel-rbe:
           command: bazel build //...
-  
+  code-analysis-pr:
```

Comment:
let's call it `build-analysis-pr` so that it still uses the `build`, `test`, `deploy` terminology?

Reply:
ok